integration-cli/requirements.go

```diff
@@ -25,6 +30,34 @@ var (
 		func() bool { return supportsExec },
 		"Test requires 'docker exec' capabilities on the tested daemon.",
 	}
+	NativeExecDriver = TestRequirement{
+		func() bool {
+			if daemonExecDriver == "" {
+				// get daemon info
+				body, err := sockRequest("GET", "/info", nil)
+				if err != nil {
+					log.Fatalf("sockRequest failed for /info: %v", err)
+				}
+
+				var infoJSON map[string]interface{}
```

Comment:
@jfrazelle I think it's better if you make it:

```Go
var infoJSON struct {
    ExecutionDriver string
}
```

That way, the Unmarshal will fail if it's not there, or if it's not a string.